examples/only-logs-otel/rendered_manifests/configmap-agent.yaml

```diff
@@ -231,6 +231,8 @@ data:
           to: body
           type: move
         poll_interval: 200ms
+        retry_on_failure:
```

Comment:
If contrib 0.76.3 is needed to provide this option do you mind updating the image to 0.76.0? I was about ready to for a release.

Reply:
sry, disregard will open the PR

Reply:
I see the docker image already published https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags&tag=latest and I picked it up in https://github.com/signalfx/splunk-otel-collector-chart/pull/763

Reply:
ha, missed https://github.com/signalfx/splunk-otel-collector-chart/pull/763 while testing that image...